osgtest/tests/test_401_scitokens.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+import pwd
+import time
+
+from urllib import error, request
+
+import osgtest.library.core as core
+import osgtest.library.files as files
+import osgtest.library.osgunittest as osgunittest
+
+# Headers so that heroku doesn't block us
+HEADERS = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_3) AppleWebKit/537.36 (KHTML, like Gecko)' +
```

Comment:
lol really?

Reply:
I'm not sure actually but it was in @djw8605's script that he passed on for requesting tokens so it's going to cargo cult its way in here :). Please don't ask me to kick off a take 23 of the tests!